CGMBLEKitUI/TransmitterSettingsViewController.swift

```diff
@@ -115,7 +115,9 @@ class TransmitterSettingsViewController: UITableViewController {
     }
 
     private enum AgeRow: Int, CaseIterable {
-        case sensor
+        case sensorage
```

Comment:
camel case and full words: `sensorAge`, `sensorCountdown`, `sensorExpirationDate`

Reply:
Done.